wordpress/wp-content/plugins/cds-base/classes/Modules/Users/Users.php

```diff
@@ -68,7 +68,12 @@ public function getRoles()
             "This role is allows the user to write and publish articles online to the collection.",
             "cds-snc"
         );
-        $roleDescriptions = ["administrator" => $administrator, "gceditor" => $gceditor];
+
+        $gcwriter = __(
+            "This role is allows the user to write articles online to the collection.",
+            "cds-snc"
+        );
```

Comment:
- don't need the "is" in both descriptions
- we should probably change this to not say "collection"?

Also, the main difference between this one and an Editor is that this one can't publish, right? Should that be more clear in the description? ("This role allows the user to draft articles but not to publish them.")